osconfig_tests/osconfig_server/assignment_wrapper.go

```diff
@@ -0,0 +1,76 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// Package osconfigserver contains wrapper around osconfig service APIs and helper methods
+package osconfigserver
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+
+	osconfigpb "github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/google-osconfig-agent/_internal/gapi-cloud-osconfig-go/google.golang.org/genproto/googleapis/cloud/osconfig/v1alpha1"
+)
+
+type Assignment struct {
+	*osconfigpb.Assignment
+}
+
+// CreateAssignment is a wrapper around createAssignment API
+func CreateAssignment(ctx context.Context, logger *log.Logger, assignment *Assignment, parent string) (*Assignment, error) {
```

Comment:
I don't think these functions should take a logger function, the logging should just be done by the caller.

Reply:
yeah, i agree, it should not. i added them while initially writing the wrapper package. it helped me debugging the apis.

will remove logger from the wrapper package.